test/config/credentials_default.json

```diff
@@ -19,8 +20,8 @@
     ],
     "ora": [
       {
-        "username": "vagrant",
-        "password": "vagrant"
+        "username": "onrack",
+        "password": "onrack"
```

Comment:
Is this right with the default still being "vagrant"? (just curious, because it would be easy to be testing against a stack and miss changing this back!)